### PR TITLE
Fix #149: bug in codeowner message

### LINKED ISSUE #149: Bug in owners message
Oppiabot's "new owners" message is confusing, see: https://github.com/oppia/oppia/pull/11084#issuecomment-719022812. Specifically:
1. It's copying unrelated things from the owners file (comments)
2. Why does Vinita care about this? Just because she's the initial reviewer? Sandeep actually has to approve the change, so it seems more relevant to him as the codeowner of the changed file.

### PR DESCRIPTION
**Pull request by HarshKhilawala**
## Explanation

Fix #149 
Fixes bug in codeowner message. 

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by HarshKhilawala** [comment]
@vojtechjelinek @jameesjohn PTAL!

File: userWhitelist.json
Comment (vojtechjelinek) [reply userWhitelist.json:25]: Why we need constant? Why don't we retrieve this from the CODEOWNERS file?

File: lib/checkForNewCodeowner.js
Comment (vojtechjelinek) [inline lib/checkForNewCodeowner.js:69]:
```suggestion
    return (
      change.startsWith(newAddition) && !change.startsWith(newCommentAddition)
    );
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: userWhitelist.json
Comment (HarshKhilawala) [reply userWhitelist.json:25]: We decided to go this way because we felt it was better to keep it constant as this usually doesn't change for a long period of time. Also, it would be easier to reference this for future purposes too. @vojtechjelinek

File: lib/checkForNewCodeowner.js
Comment (HarshKhilawala) [reply lib/checkForNewCodeowner.js:69]: Done!

**Comment by HarshKhilawala** [comment]
@vojtechjelinek PTAL

File: userWhitelist.json
Comment (vojtechjelinek) [reply userWhitelist.json]: Ok.

File: lib/checkForNewCodeowner.js
Comment (vojtechjelinek) [inline lib/checkForNewCodeowner.js:71]: We have this twice in this PR, should we separate it into its own function and use it in both places?

File: lib/checkForNewCodeowner.js
Comment (vojtechjelinek) [inline lib/checkForNewCodeowner.js:136]:
```suggestion
      // Ping the code owner of CODEOWNERS file since we just want to flag that a
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Two additional comments.

File: lib/checkForNewCodeowner.js
Comment (HarshKhilawala) [reply lib/checkForNewCodeowner.js]: Done!

File: lib/checkForNewCodeowner.js
Comment (HarshKhilawala) [reply lib/checkForNewCodeowner.js]: Created the function!

**Comment by HarshKhilawala** [comment]
@vojtechjelinek Let me know if there are some necessary changes to be made

**Review by vojtechjelinek — APPROVED** [review]
LGTM! Let's wait for a review from @jameesjohn and we can merge this.

**Review by jameesjohn — APPROVED** [review]
LGTM!
Thanks, @HarshKhilawala.

**Pull request merged**
